packages/react/package.json

```diff
@@ -29,7 +29,7 @@
     "clean": "rimraf es lib umd storybook-static",
     "prepublish": "yarn build",
     "start": "yarn storybook",
-    "storybook": "start-storybook -p 9000"
+    "storybook": "rimraf node_modules/.cache/storybook && start-storybook -p 9000"
```

Comment:
Should we delete the storybook cache every time? Does it significantly slow down subsequent start up I think right?

Reply:
Ran into a gnarly issue where storybook caches `.storybook` and won't conditionally load panels no matter what I tried 😞 This shouldn't impact start-up time, at least as far as I could tell